python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -365,21 +371,22 @@ def kill(self):
         if not self.id:
             return None
         dirac_cmd = 'kill(%d)' % self.id
-        result = execute(dirac_cmd)
-        if not result_ok(result):
-            raise BackendError('Dirac', 'Could not kill job: %s' % str(result))
-        return result['OK']
+        try:
+            result = execute(dirac_cmd)
+        except GangaDiracException as err:
+            raise BackendError('Dirac', 'Could not kill job: %s' % str(err))
```

Comment:
`BackendError` is expected from this interface is a `GangaDiracError` has been thrown the rest of Ganga probably does not expect to see a `GangaDiracError` here so I'd rather submit a sensible `BackendError` which could be raised from any backend.